action.yml

```diff
@@ -9,12 +9,8 @@ branding:
 inputs:
   pkl-version:
     description: The Pkl version to install.
-    required: true
-  github-token:
-    description:
-      A GitHub token to use for fetching the GitHub release asset from
-      apple/pkl.
-    default: ${{ github.token }}
+    required: false
+    default: '0.26.3'
```

Comment:
Setting a default value for Pkl version here feels risky, even though I know it will make using the action less verbose. That said, it introduces some complications, namely: what _should_ the default value be? If Pkl introduces a new version, are we obligated to update this (creating maintenance overhead)?
But most importantly, Pkl is in a pre-1.0 phase, which means each new version could include source-breaking changes. If we were to update the silent default to a new Pkl version that had source-breaking changes, we would invisibly break everyone's pipelines who was using the default version.

I think until Pkl releases a stable 1.0, or some kind of "LTS" version (that won't have source-breaking changes in the future), it's better to require users to manually specify the Pkl version and therefore have to actively engage in the choice (to make sure the version here matches what they use locally to develop).

Reply:
Agreed, this makes a lot of sense